packages/rancher-sriov/generated-changes/overlay/templates/certmanagercerts.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.operator.enableAdmissionController -}}
+{{- if .Values.cert_manager -}}
+{{- if not (.Capabilities.APIVersions.Has "cert-manager.io/v1") -}}
```

Comment:
Are these intentionally separated or could we use an `and` here to omit having three `end` blocks?

Reply:
good idea. I can remove one end combining the first two